### PR TITLE
Update UG with glossary section

### PR DESCRIPTION
**Pull request by WangCheng0116**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -6,30 +6,31 @@ title: User Guide
 Flashlingo is a **desktop app for learning words by flash cards, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). Flashlingo is designed with beginner language learners in mind, specialising in helping them expand their vocabulary.
 
 * Table of Contents
-{:toc}
+  {:toc}
```
Comment (A1WAYSD) [inline docs/UserGuide.md:9]: Please do not add the space or else toc won't be automatically generated on web page. Thanks!

File: docs/UserGuide.md [hunk 2]
```diff
@@ -354,7 +357,7 @@ Then, use the `load` command to import the data.
 | **Help**                | `help`                                                                                                                     |
 | **List**                | `list`                                                                                                                     |
 | **Add**                 | `add w/<WORD> t/<TRANSLATION> [wl/<WORD_LANGUAGE>] [tl/<TRANSLATION_LANGUAGE>]` <br> e.g., `add w/regarder t/look`         |
-| **Delete**              | `delete <Index>`<br> e.g., `delete 1`                                                                                      |
+| **Delete**              | `delete <Index>`<br> e.g., `del 1`                                                                                         |
```
Comment (A1WAYSD) [inline docs/UserGuide.md:360]: Currently the app supports "delete" but not "del"?

**Review by A1WAYSD — COMMENTED** [review]
Thanks for correcting all the indexes and adding the glossary! Some minor issues otherwise LGTM!

File: docs/UserGuide.md (hunk 2 above)
Comment (WangCheng0116) [reply docs/UserGuide.md]: Fixed :)

File: docs/UserGuide.md (hunk 1 above)
Comment (WangCheng0116) [reply docs/UserGuide.md]: Fixed :)

**Review by Song-Mengfei — APPROVED** [review]

**Pull request merged**